### DISCUSSION TITLE
RSS feeds Dashboard Widget - increase subject text length

### DISCUSSION THREAD

**Discussion by PercyP** [YetiForceCompany/YetiForceCRM#15294] [How to] [answered]
Hi,

does anyone know how to increase the length of text to display in the RSS Dashboard widget? I have gone through settings but cannot see anything that relates to this. 

Kind regards

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://github.com/YetiForceCompany/YetiForceCRM/blob/d900ebe3f9eed9779add3d559f41bc1342f91cde/modules/Vtiger/dashboards/Rss.php#L43

**Comment by PercyP** [comment]
Perfect! Thankyou!
